packages/module/src/DetailsPageHeader/utils/ActionButtons.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button, Flex, FlexItem, Tooltip } from '@patternfly/react-core';
+import React from 'react';
+
+export interface ActionButtonProps {
+  /** Text rendered inside the button */
+  label: string;
+  /** A callback for when the action button is clicked */
+  callback: (event: React.MouseEvent) => void;
+  /** Adds button variant styles */
+  variant?: 'primary' | 'secondary' | 'tertiary' | 'link';
+  /** ID for the action button */
+  id?: string;
```

Comment:
I see this as an ID for the tooltip not the button.

Reply:
Updated.